### PR TITLE
[Feature] : Way to prevent auto-scroll down in "Container logs"

### PR DESCRIPTION
**Pull request by collin5**
PR enables the user to disable or enable following(auto-scroll down) for container logs. Enabled by default but can be disabled on user demand 🙂 .

_[screenshot]_
screen shot 2018-01-03 at 11 12 42
 
Closes issue #2611

### CODE REVIEW

**Comment by collin5** [comment]
Any feedback on this @FrenchBen. Thanks

File: src/components/ContainerHomeLogs.react.js [hunk 1]
```diff
@@ -40,17 +40,20 @@ let prevBottom = 0;
...
+    let $target = event.target;
+    this.setState((prevState)=>({
+      fontSize: $target.value,
+      follow: prevState.follow
+    }));
   },
   componentDidUpdate: function () {
     var node = $('.logs').get()[0];
-    node.scrollTop = node.scrollHeight;
+    node.scrollTop = (this.state.follow)? node.scrollHeight: 0;
```
Comment (FrenchBen) [inline src/components/ContainerHomeLogs.react.js]: As soon as I "unfollow" I end up being thrown to the top of the logs - Anyway to capture the current scroll position and simply keep the user there?

Comment (collin5) [reply src/components/ContainerHomeLogs.react.js]: Thanks, hadn't thought of this before. Going to fix this ASAP

**Comment by collin5** [comment]
Thanks for the feedback @FrenchBen. Got this fixed 👍

**Pull request merged**